src/cluster_asm.c

```diff
@@ -1407,14 +1423,25 @@ void asmStartImportTask(asmTask *task) {
     if (task->operation != ASM_IMPORT || task->state != ASM_NONE) return;
     sds slot_ranges_str = slotRangeArrayToString(task->slot_ranges);
 
+    /* Cannot start import task since pause action is performed. Otherwise, we will
+     * break the promise that no writes are performed during the pause. */
+    if (isPausedActions(PAUSE_ACTION_CLIENT_ALL) ||
+        isPausedActions(PAUSE_ACTION_CLIENT_WRITE))
+    {
+        serverLog(LL_DEBUG, "Can not start import task for slots: %s since server is paused",
```

Comment:
Do you think we should use LL_NOTICE level at least? Because this is the only place to figure out why migration did not start. Though, we are going to flood the log if write pause stays longer. So, I'm not sure which one is better. Maybe LL_NOTICE is not necessary.

Reply:
i was worried this may print too many logs. so let print this log every 5s